### ISSUE TITLE
mysanantonio.com

### ISSUE THREAD

**Issue by terrorist96** [uBlockOrigin/uAssets#3225]
### URL(s) where the issue occurs

`https://www.mysanantonio.com/news/local/article/Seven-life-sentences-for-ex-Zetas-boss-who-took-11253299.php#item-85307-tbla-5`
`https://www.mysanantonio.com/news/local/article/Seven-life-sentences-for-ex-Zetas-boss-who-took-11253299.php`

### Describe the issue

Sponsored ads getting through. Infinite loading I do have uBO Extra installed.

### Screenshot(s)

![image](https://user-images.githubusercontent.com/12802056/44092204-bd7729b6-9f9d-11e8-8df3-38fe1a26ffb8.png)

### Versions

- Browser/version: Chrome 68
- uBlock Origin version: 1.16.14

### Settings

Default

**Comment by mapx-** [comment]
well, g00 crap ..

**Comment by terrorist96** [comment]
Yep. Wasn't sure if I should report here or there. I opened an issue there too. https://github.com/gorhill/uBO-Extra/issues/97
